[2019-03-08 14:05:40] php.INFO: User Deprecated: Auto-registration of the command "Symfony\Bundle\SwiftmailerBundle\Command\NewEmailCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Symfony\\Bundle\\SwiftmailerBundle\\Command\\NewEmailCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:05:40] php.INFO: User Deprecated: Auto-registration of the command "Symfony\Bundle\SwiftmailerBundle\Command\SendEmailCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Symfony\\Bundle\\SwiftmailerBundle\\Command\\SendEmailCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:05:40] php.INFO: User Deprecated: Auto-registration of the command "Symfony\Bundle\SwiftmailerBundle\Command\DebugCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Symfony\\Bundle\\SwiftmailerBundle\\Command\\DebugCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:05:40] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateBundleCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateBundleCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:05:40] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateDoctrineCrudCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateDoctrineCrudCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:05:40] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateDoctrineFormCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateDoctrineFormCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:05:40] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateDoctrineEntityCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateDoctrineEntityCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:05:40] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateControllerCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateControllerCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:05:40] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateCommandCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateCommandCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:08:08] php.INFO: User Deprecated: Auto-registration of the command "Symfony\Bundle\SwiftmailerBundle\Command\NewEmailCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Symfony\\Bundle\\SwiftmailerBundle\\Command\\NewEmailCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:08:08] php.INFO: User Deprecated: Auto-registration of the command "Symfony\Bundle\SwiftmailerBundle\Command\SendEmailCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Symfony\\Bundle\\SwiftmailerBundle\\Command\\SendEmailCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:08:08] php.INFO: User Deprecated: Auto-registration of the command "Symfony\Bundle\SwiftmailerBundle\Command\DebugCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Symfony\\Bundle\\SwiftmailerBundle\\Command\\DebugCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:08:08] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateBundleCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateBundleCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:08:08] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateDoctrineCrudCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateDoctrineCrudCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:08:08] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateDoctrineFormCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateDoctrineFormCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:08:08] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateDoctrineEntityCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateDoctrineEntityCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:08:08] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateControllerCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateControllerCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:08:08] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateCommandCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateCommandCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:08:08] php.DEBUG: Warning: file(/proc/mounts): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Command/CacheClearCommand.php\",\"line\":148,\"trace\":[{\"file\":\"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Console/Command/Command.php\",\"line\":255,\"function\":\"execute\",\"class\":\"Symfony\\\\Bundle\\\\FrameworkBundle\\\\Command\\\\CacheClearCommand\",\"type\":\"->\"}],\"count\":1})"} []
[2019-03-08 14:08:10] php.INFO: User Deprecated: Auto-registration of the command "Symfony\Bundle\SwiftmailerBundle\Command\NewEmailCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Symfony\\Bundle\\SwiftmailerBundle\\Command\\NewEmailCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:08:10] php.INFO: User Deprecated: Auto-registration of the command "Symfony\Bundle\SwiftmailerBundle\Command\SendEmailCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Symfony\\Bundle\\SwiftmailerBundle\\Command\\SendEmailCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:08:10] php.INFO: User Deprecated: Auto-registration of the command "Symfony\Bundle\SwiftmailerBundle\Command\DebugCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Symfony\\Bundle\\SwiftmailerBundle\\Command\\DebugCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:08:10] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateBundleCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateBundleCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:08:10] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateDoctrineCrudCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateDoctrineCrudCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:08:10] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateDoctrineFormCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateDoctrineFormCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:08:10] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateDoctrineEntityCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateDoctrineEntityCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:08:10] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateControllerCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateControllerCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:08:10] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateCommandCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateCommandCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:18:53] php.INFO: User Deprecated: Auto-registration of the command "Symfony\Bundle\SwiftmailerBundle\Command\NewEmailCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Symfony\\Bundle\\SwiftmailerBundle\\Command\\NewEmailCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:18:53] php.INFO: User Deprecated: Auto-registration of the command "Symfony\Bundle\SwiftmailerBundle\Command\SendEmailCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Symfony\\Bundle\\SwiftmailerBundle\\Command\\SendEmailCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:18:53] php.INFO: User Deprecated: Auto-registration of the command "Symfony\Bundle\SwiftmailerBundle\Command\DebugCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Symfony\\Bundle\\SwiftmailerBundle\\Command\\DebugCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:18:53] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateBundleCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateBundleCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:18:53] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateDoctrineCrudCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateDoctrineCrudCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:18:53] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateDoctrineFormCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateDoctrineFormCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:18:53] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateDoctrineEntityCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateDoctrineEntityCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:18:53] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateControllerCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateControllerCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:18:53] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateCommandCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateCommandCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:20:08] php.INFO: User Deprecated: Auto-registration of the command "Symfony\Bundle\SwiftmailerBundle\Command\NewEmailCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Symfony\\Bundle\\SwiftmailerBundle\\Command\\NewEmailCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:20:08] php.INFO: User Deprecated: Auto-registration of the command "Symfony\Bundle\SwiftmailerBundle\Command\SendEmailCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Symfony\\Bundle\\SwiftmailerBundle\\Command\\SendEmailCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:20:08] php.INFO: User Deprecated: Auto-registration of the command "Symfony\Bundle\SwiftmailerBundle\Command\DebugCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Symfony\\Bundle\\SwiftmailerBundle\\Command\\DebugCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:20:08] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateBundleCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateBundleCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:20:08] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateDoctrineCrudCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateDoctrineCrudCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:20:08] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateDoctrineFormCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateDoctrineFormCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:20:08] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateDoctrineEntityCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateDoctrineEntityCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:20:08] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateControllerCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateControllerCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:20:08] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateCommandCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateCommandCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:20:08] console.ERROR: Error thrown while running command "doctrine:schema:update -force". Message: "The "-o" option does not exist." {"exception":"[object] (Symfony\\Component\\Console\\Exception\\RuntimeException(code: 0): The \"-o\" option does not exist. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Console/Input/ArgvInput.php:125)","command":"doctrine:schema:update -force","message":"The \"-o\" option does not exist."} []
[2019-03-08 14:20:08] console.DEBUG: Command "doctrine:schema:update -force" exited with code "1" {"command":"doctrine:schema:update -force","code":1} []
[2019-03-08 14:20:24] php.INFO: User Deprecated: Auto-registration of the command "Symfony\Bundle\SwiftmailerBundle\Command\NewEmailCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Symfony\\Bundle\\SwiftmailerBundle\\Command\\NewEmailCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:20:24] php.INFO: User Deprecated: Auto-registration of the command "Symfony\Bundle\SwiftmailerBundle\Command\SendEmailCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Symfony\\Bundle\\SwiftmailerBundle\\Command\\SendEmailCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:20:24] php.INFO: User Deprecated: Auto-registration of the command "Symfony\Bundle\SwiftmailerBundle\Command\DebugCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Symfony\\Bundle\\SwiftmailerBundle\\Command\\DebugCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:20:24] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateBundleCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateBundleCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:20:24] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateDoctrineCrudCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateDoctrineCrudCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:20:24] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateDoctrineFormCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateDoctrineFormCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:20:24] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateDoctrineEntityCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateDoctrineEntityCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:20:24] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateControllerCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateControllerCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:20:24] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateCommandCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateCommandCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:20:24] doctrine.DEBUG: SHOW FULL TABLES WHERE Table_type = 'BASE TABLE' [] []
[2019-03-08 14:20:24] doctrine.DEBUG: SHOW FULL TABLES WHERE Table_type = 'BASE TABLE' [] []
[2019-03-08 14:20:24] doctrine.DEBUG: CREATE TABLE fos_user (id INT AUTO_INCREMENT NOT NULL, username VARCHAR(180) NOT NULL, username_canonical VARCHAR(180) NOT NULL, email VARCHAR(180) NOT NULL, email_canonical VARCHAR(180) NOT NULL, enabled TINYINT(1) NOT NULL, salt VARCHAR(255) DEFAULT NULL, password VARCHAR(255) NOT NULL, last_login DATETIME DEFAULT NULL, confirmation_token VARCHAR(180) DEFAULT NULL, password_requested_at DATETIME DEFAULT NULL, roles LONGTEXT NOT NULL COMMENT '(DC2Type:array)', UNIQUE INDEX UNIQ_957A647992FC23A8 (username_canonical), UNIQUE INDEX UNIQ_957A6479A0D96FBF (email_canonical), UNIQUE INDEX UNIQ_957A6479C05FB297 (confirmation_token), PRIMARY KEY(id)) DEFAULT CHARACTER SET UTF8 COLLATE UTF8_unicode_ci ENGINE = InnoDB [] []
[2019-03-08 14:21:23] php.INFO: User Deprecated: Auto-registration of the command "Symfony\Bundle\SwiftmailerBundle\Command\NewEmailCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Symfony\\Bundle\\SwiftmailerBundle\\Command\\NewEmailCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:21:23] php.INFO: User Deprecated: Auto-registration of the command "Symfony\Bundle\SwiftmailerBundle\Command\SendEmailCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Symfony\\Bundle\\SwiftmailerBundle\\Command\\SendEmailCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:21:23] php.INFO: User Deprecated: Auto-registration of the command "Symfony\Bundle\SwiftmailerBundle\Command\DebugCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Symfony\\Bundle\\SwiftmailerBundle\\Command\\DebugCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:21:23] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateBundleCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateBundleCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:21:23] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateDoctrineCrudCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateDoctrineCrudCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:21:23] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateDoctrineFormCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateDoctrineFormCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:21:23] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateDoctrineEntityCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateDoctrineEntityCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:21:23] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateControllerCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateControllerCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:21:23] php.INFO: User Deprecated: Auto-registration of the command "Sensio\Bundle\GeneratorBundle\Command\GenerateCommandCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Sensio\\Bundle\\GeneratorBundle\\Command\\GenerateCommandCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:177)"} []
[2019-03-08 14:22:16] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /register"" at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 139 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /register\" at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/EventListener/RouterListener.php:139, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/var/cache/dev/appDevDebugProjectContainerUrlMatcher.php:129)"} []
[2019-03-08 14:22:17] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"385d1c","_route":"_wdt"},"request_uri":"http://localhost:8888/2018-l3ac1/branches/Publish/web/app_dev.php/_wdt/385d1c","method":"GET"} []
[2019-03-08 14:22:59] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Config\Exception\FileLoaderLoadException: "The definition of "fos_user" in "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml")." at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php line 168 {"exception":"[object] (Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:168, InvalidArgumentException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:203)"} []
[2019-03-08 14:23:00] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The definition of "fos_user" in "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml")."). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Bundle/TwigBundle/Resources/views/Exception/trace.html.twig line 13) {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml\").\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Bundle/TwigBundle/Resources/views/Exception/trace.html.twig:13, Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:168, InvalidArgumentException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:203)"} []
[2019-03-08 14:23:00] php.CRITICAL: Uncaught Exception: An exception has been thrown during the rendering of a template ("The definition of "fos_user" in "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml")."). {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml\").\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Bundle/TwigBundle/Resources/views/Exception/trace.html.twig:13, Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:168, InvalidArgumentException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:203, Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:168, InvalidArgumentException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:203)"} []
[2019-03-08 14:23:00] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("The definition of "fos_user" in "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml").")." at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Bundle/TwigBundle/Resources/views/Exception/trace.html.twig line 13 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml\").\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Bundle/TwigBundle/Resources/views/Exception/trace.html.twig:13, Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:168, InvalidArgumentException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:203, Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:168, InvalidArgumentException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:203)"} []
[2019-03-08 14:23:00] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The definition of "fos_user" in "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml")."). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Bundle/TwigBundle/Resources/views/Exception/trace.html.twig line 13) {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml\").\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Bundle/TwigBundle/Resources/views/Exception/trace.html.twig:13, Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:168, InvalidArgumentException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:203)"} []
[2019-03-08 14:23:25] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Config\Exception\FileLoaderLoadException: "The definition of "fos_user" in "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml (which is being imported from "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml")." at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php line 168 {"exception":"[object] (Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:168, InvalidArgumentException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml\" must be a YAML array. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:203)"} []
[2019-03-08 14:23:25] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The definition of "fos_user" in "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml (which is being imported from "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml")."). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Bundle/TwigBundle/Resources/views/Exception/trace.html.twig line 13) {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\").\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Bundle/TwigBundle/Resources/views/Exception/trace.html.twig:13, Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:168, InvalidArgumentException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml\" must be a YAML array. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:203)"} []
[2019-03-08 14:23:25] php.CRITICAL: Uncaught Exception: An exception has been thrown during the rendering of a template ("The definition of "fos_user" in "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml (which is being imported from "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml")."). {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\").\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Bundle/TwigBundle/Resources/views/Exception/trace.html.twig:13, Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:168, InvalidArgumentException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml\" must be a YAML array. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:203, Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:168, InvalidArgumentException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml\" must be a YAML array. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:203)"} []
[2019-03-08 14:23:25] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("The definition of "fos_user" in "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml (which is being imported from "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml").")." at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Bundle/TwigBundle/Resources/views/Exception/trace.html.twig line 13 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\").\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Bundle/TwigBundle/Resources/views/Exception/trace.html.twig:13, Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:168, InvalidArgumentException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml\" must be a YAML array. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:203, Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:168, InvalidArgumentException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml\" must be a YAML array. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:203)"} []
[2019-03-08 14:23:25] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The definition of "fos_user" in "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml (which is being imported from "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml")."). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Bundle/TwigBundle/Resources/views/Exception/trace.html.twig line 13) {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\").\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Bundle/TwigBundle/Resources/views/Exception/trace.html.twig:13, Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:168, InvalidArgumentException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/src/ConnexionBundle/Resources/config/routing.yml\" must be a YAML array. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:203)"} []
[2019-03-08 14:24:35] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Config\Exception\FileLoaderLoadException: "The definition of "fos_user" in "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml")." at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php line 168 {"exception":"[object] (Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:168, InvalidArgumentException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:203)"} []
[2019-03-08 14:24:35] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The definition of "fos_user" in "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml")."). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Bundle/TwigBundle/Resources/views/Exception/trace.html.twig line 13) {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml\").\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Bundle/TwigBundle/Resources/views/Exception/trace.html.twig:13, Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:168, InvalidArgumentException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:203)"} []
[2019-03-08 14:24:35] php.CRITICAL: Uncaught Exception: An exception has been thrown during the rendering of a template ("The definition of "fos_user" in "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml")."). {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml\").\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Bundle/TwigBundle/Resources/views/Exception/trace.html.twig:13, Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:168, InvalidArgumentException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:203, Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:168, InvalidArgumentException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:203)"} []
[2019-03-08 14:24:36] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("The definition of "fos_user" in "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml").")." at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Bundle/TwigBundle/Resources/views/Exception/trace.html.twig line 13 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml\").\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Bundle/TwigBundle/Resources/views/Exception/trace.html.twig:13, Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:168, InvalidArgumentException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:203, Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:168, InvalidArgumentException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:203)"} []
[2019-03-08 14:24:36] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("The definition of "fos_user" in "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from "/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml")."). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Bundle/TwigBundle/Resources/views/Exception/trace.html.twig line 13) {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml\").\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Bundle/TwigBundle/Resources/views/Exception/trace.html.twig:13, Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array in /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml (which is being imported from \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing_dev.yml\"). at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Config/Loader/FileLoader.php:168, InvalidArgumentException(code: 0): The definition of \"fos_user\" in \"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/app/config/routing.yml\" must be a YAML array. at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Routing/Loader/YamlFileLoader.php:203)"} []
[2019-03-08 14:31:19] request.INFO: Matched route "fos_user_registration_register". {"route":"fos_user_registration_register","route_parameters":{"_controller":"Symfony\\Bundle\\FrameworkBundle\\Controller\\RedirectController::urlRedirectAction","_route":"fos_user_registration_register","path":"/register/","permanent":true,"scheme":null,"httpPort":8888,"httpsPort":443},"request_uri":"http://localhost:8888/2018-l3ac1/branches/Publish/web/app_dev.php/register","method":"GET"} []
[2019-03-08 14:31:19] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-03-08 14:31:19] request.INFO: Matched route "fos_user_registration_register". {"route":"fos_user_registration_register","route_parameters":{"_controller":"fos_user.registration.controller:registerAction","_route":"fos_user_registration_register"},"request_uri":"http://localhost:8888/2018-l3ac1/branches/Publish/web/app_dev.php/register/","method":"GET"} []
[2019-03-08 14:31:19] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-03-08 14:31:20] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"2b47e4","_route":"_wdt"},"request_uri":"http://localhost:8888/2018-l3ac1/branches/Publish/web/app_dev.php/_wdt/2b47e4","method":"GET"} []
[2019-03-08 14:31:50] request.INFO: Matched route "fos_user_registration_register". {"route":"fos_user_registration_register","route_parameters":{"_controller":"fos_user.registration.controller:registerAction","_route":"fos_user_registration_register"},"request_uri":"http://localhost:8888/2018-l3ac1/branches/Publish/web/app_dev.php/register/","method":"POST"} []
[2019-03-08 14:31:50] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-03-08 14:31:50] doctrine.DEBUG: SELECT t0.username AS username_1, t0.username_canonical AS username_canonical_2, t0.email AS email_3, t0.email_canonical AS email_canonical_4, t0.enabled AS enabled_5, t0.salt AS salt_6, t0.password AS password_7, t0.last_login AS last_login_8, t0.confirmation_token AS confirmation_token_9, t0.password_requested_at AS password_requested_at_10, t0.roles AS roles_11, t0.id AS id_12 FROM fos_user t0 WHERE t0.username_canonical = ? ["juju"] []
[2019-03-08 14:31:50] doctrine.DEBUG: SELECT t0.username AS username_1, t0.username_canonical AS username_canonical_2, t0.email AS email_3, t0.email_canonical AS email_canonical_4, t0.enabled AS enabled_5, t0.salt AS salt_6, t0.password AS password_7, t0.last_login AS last_login_8, t0.confirmation_token AS confirmation_token_9, t0.password_requested_at AS password_requested_at_10, t0.roles AS roles_11, t0.id AS id_12 FROM fos_user t0 WHERE t0.email_canonical = ? ["juju@gmail.com"] []
[2019-03-08 14:31:50] request.CRITICAL: Uncaught PHP Exception RuntimeException: "No encoder has been configured for account "ConnexionBundle\Entity\User"." at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Security/Core/Encoder/EncoderFactory.php line 51 {"exception":"[object] (RuntimeException(code: 0): No encoder has been configured for account \"ConnexionBundle\\Entity\\User\". at /Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/symfony/symfony/src/Symfony/Component/Security/Core/Encoder/EncoderFactory.php:51)"} []
[2019-03-08 14:31:51] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"bdd479","_route":"_wdt"},"request_uri":"http://localhost:8888/2018-l3ac1/branches/Publish/web/app_dev.php/_wdt/bdd479","method":"GET"} []
[2019-03-08 14:32:58] request.INFO: Matched route "fos_user_registration_register". {"route":"fos_user_registration_register","route_parameters":{"_controller":"Symfony\\Bundle\\FrameworkBundle\\Controller\\RedirectController::urlRedirectAction","_route":"fos_user_registration_register","path":"/register/","permanent":true,"scheme":null,"httpPort":8888,"httpsPort":443},"request_uri":"http://localhost:8888/2018-l3ac1/branches/Publish/web/app_dev.php/register","method":"GET"} []
[2019-03-08 14:32:58] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-03-08 14:32:58] request.INFO: Matched route "fos_user_registration_register". {"route":"fos_user_registration_register","route_parameters":{"_controller":"Symfony\\Bundle\\FrameworkBundle\\Controller\\RedirectController::urlRedirectAction","_route":"fos_user_registration_register","path":"/register/","permanent":true,"scheme":null,"httpPort":8888,"httpsPort":443},"request_uri":"http://localhost:8888/2018-l3ac1/branches/Publish/web/app_dev.php/register","method":"GET"} []
[2019-03-08 14:32:58] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-03-08 14:32:59] request.INFO: Matched route "fos_user_registration_register". {"route":"fos_user_registration_register","route_parameters":{"_controller":"fos_user.registration.controller:registerAction","_route":"fos_user_registration_register"},"request_uri":"http://localhost:8888/2018-l3ac1/branches/Publish/web/app_dev.php/register/","method":"GET"} []
[2019-03-08 14:32:59] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-03-08 14:32:59] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"3bb6ac","_route":"_wdt"},"request_uri":"http://localhost:8888/2018-l3ac1/branches/Publish/web/app_dev.php/_wdt/3bb6ac","method":"GET"} []
[2019-03-08 14:33:34] request.INFO: Matched route "fos_user_registration_register". {"route":"fos_user_registration_register","route_parameters":{"_controller":"fos_user.registration.controller:registerAction","_route":"fos_user_registration_register"},"request_uri":"http://localhost:8888/2018-l3ac1/branches/Publish/web/app_dev.php/register/","method":"POST"} []
[2019-03-08 14:33:34] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-03-08 14:33:35] doctrine.DEBUG: SELECT t0.username AS username_1, t0.username_canonical AS username_canonical_2, t0.email AS email_3, t0.email_canonical AS email_canonical_4, t0.enabled AS enabled_5, t0.salt AS salt_6, t0.password AS password_7, t0.last_login AS last_login_8, t0.confirmation_token AS confirmation_token_9, t0.password_requested_at AS password_requested_at_10, t0.roles AS roles_11, t0.id AS id_12 FROM fos_user t0 WHERE t0.username_canonical = ? ["juju"] []
[2019-03-08 14:33:35] doctrine.DEBUG: SELECT t0.username AS username_1, t0.username_canonical AS username_canonical_2, t0.email AS email_3, t0.email_canonical AS email_canonical_4, t0.enabled AS enabled_5, t0.salt AS salt_6, t0.password AS password_7, t0.last_login AS last_login_8, t0.confirmation_token AS confirmation_token_9, t0.password_requested_at AS password_requested_at_10, t0.roles AS roles_11, t0.id AS id_12 FROM fos_user t0 WHERE t0.email_canonical = ? ["juju@gmail.com"] []
[2019-03-08 14:33:37] doctrine.DEBUG: "START TRANSACTION" [] []
[2019-03-08 14:33:37] doctrine.DEBUG: INSERT INTO fos_user (username, username_canonical, email, email_canonical, enabled, salt, password, last_login, confirmation_token, password_requested_at, roles) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) {"1":"juju","2":"juju","3":"juju@gmail.com","4":"juju@gmail.com","5":false,"6":null,"7":"$2y$13$kgJXnW9Y.xZJnn75e.b [...]","8":null,"9":"NbED4oMew_UpC4Dai02SYhpgCG [...]","10":null,"11":[]} []
[2019-03-08 14:33:37] doctrine.DEBUG: "COMMIT" [] []
[2019-03-08 14:33:37] php.DEBUG: Warning: stream_socket_client(): unable to connect to 127.0.0.1:25 (Connection refused) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/swiftmailer/swiftmailer/lib/classes/Swift/Transport/StreamBuffer.php\",\"line\":275,\"trace\":[{\"file\":\"/Applications/MAMP/htdocs/2018-l3ac1/branches/Publish/vendor/swiftmailer/swiftmailer/lib/classes/Swift/Transport/StreamBuffer.php\",\"line\":62,\"function\":\"_establishSocketConnection\",\"class\":\"Swift_Transport_StreamBuffer\",\"type\":\"->\"}],\"count\":1})"} []
[2019-03-08 14:33:37] app.ERROR: Exception occurred while flushing email queue: Connection could not be established with host 127.0.0.1 [Connection refused #61] [] []
[2019-03-08 14:33:37] request.INFO: Matched route "fos_user_registration_check_email". {"route":"fos_user_registration_check_email","route_parameters":{"_controller":"fos_user.registration.controller:checkEmailAction","_route":"fos_user_registration_check_email"},"request_uri":"http://localhost:8888/2018-l3ac1/branches/Publish/web/app_dev.php/register/check-email","method":"GET"} []
[2019-03-08 14:33:37] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-03-08 14:33:37] doctrine.DEBUG: SELECT t0.username AS username_1, t0.username_canonical AS username_canonical_2, t0.email AS email_3, t0.email_canonical AS email_canonical_4, t0.enabled AS enabled_5, t0.salt AS salt_6, t0.password AS password_7, t0.last_login AS last_login_8, t0.confirmation_token AS confirmation_token_9, t0.password_requested_at AS password_requested_at_10, t0.roles AS roles_11, t0.id AS id_12 FROM fos_user t0 WHERE t0.email_canonical = ? LIMIT 1 ["juju@gmail.com"] []
[2019-03-08 14:33:38] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"393624","_route":"_wdt"},"request_uri":"http://localhost:8888/2018-l3ac1/branches/Publish/web/app_dev.php/_wdt/393624","method":"GET"} []
